ocular/src/chain/info.rs

```diff
@@ -138,6 +136,7 @@ impl ChainInfo {
             gas_adjustment: 1.2,
             gas_prices,
             rpc_address,
+            grpc_address: String::from(""),
```

Comment:
Structs that impl `Default` can finish their unset fields like

```rust
ChainClientConfig {
        chain_name: self.chain_name.clone(),
        account_prefix: self.bech32_prefix.clone(),
        chain_id: self.chain_id.clone(),
        gas_adjustment: 1.2,
        gas_prices,
        rpc_address,
        ..Default::default(),
}
```

In this case it's just the one field so it doesn't really matter, just worth noting.

Reply:
That's pretty cool, wasn't aware of that possibility; thanks for pointing it out